custom_components/ipx800v4/const.py

```diff
@@ -17,6 +17,7 @@
 CONF_DEFAULT_BRIGHTNESS = "default_brightness"
 CONF_ID = "id"
 CONF_IDS = "ids"
+CONF_INVERT = "invert"
```

Comment:
I think `CONF_INVERTED_VALUE` is more explicit, no ?

Reply:
Good idea, I'll rename it. I'd keep the infinitive form of the verb though, it's an action.